common/djangoapps/student/models.py

```diff
@@ -929,12 +929,15 @@ def is_course_full(self, course):
 
         return is_course_full
 
-    def users_enrolled_in(self, course_id):
-        """Return a queryset of User for every user enrolled in the course."""
-        return User.objects.filter(
-            courseenrollment__course_id=course_id,
-            courseenrollment__is_active=True
-        )
+    def users_enrolled_in(self, course_id, include_inactive_users=False):
```

Comment:
Nit: `include_inactive` as the parameter name should be sufficiently enough, I believe, without being redundant with the method name.